litespi/spi_nor_flash_module.py

```diff
@@ -0,0 +1,286 @@
+from litespi.spi_nor_features import SpiNorFeatures
+from litespi.opcodes import SpiNorFlashOpCodes
+
+
+class MetaSizes(type):
```

Comment:
Probably worth putting this in it's own file.

Remove the debugging prints.

Reply:
I removed debug prints.